src/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/GetNugetPackageBasePath.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tools.BootstrapTasks
+{
+	public class GetNugetPackageBasePath : Task
+	{
+		[Required]
+		public ITaskItem [] PackageConfigFiles { get; set; }
+
+		[Required]
+		public string PackageName { get; set; }
+
+		[Output]
+		public string BasePath { get; set; }
```

Comment:
Not sure if this is really needed? Many tasks use `string OutputDir` or such - to assign the value to a variable or an item using the `<Output>` child element of the task. What advantage would using `ITaskItem` give us over using a `string`?